classes/task.class.php

```diff
@@ -166,6 +166,7 @@ public function get_reports() {
             // webhook callback so ignore anything submitted to Turnitin in the 2 minutes.
             // Otherwise retrieve originality score if we haven't received it back within 5 minutes.
             if ($tssubmission->getstatus() == TURNITINSIM_SUBMISSION_STATUS_UPLOADED
+                && $tssubmission->get_submission_info()->status === TURNITINSIM_SUBMISSION_STATUS_COMPLETE
```

Comment:
I'm not sure this will work. The submission status is only set to COMPLETE once a score exists, and if a score exists it wouldn't get here anyway. My thoughts on this ticket was that we would have to make a call to the API using the `get_submission_info()` method to check the status.